.env.sample

```
@@ -0,0 +1,24 @@
+DJANGO_SECRET_KEY=secret_key
+
+DEBUG=True/False
+
+DATABASE_NAME=db_name
+DATABASE_USER=db_user
+DATABASE_PASSWORD=db_password
+DATABASE_HOST=db_host
+DATABASE_PORT=db_port
+
+STRIPE_API_KEY=stipe_api_key
+
+EMAIL_BACKEND=django.core.mail.backends.smtp.EmailBackend
```

для разработки я бы не прикручивал smtp бэкенд, а отправлял письма либо в файл, либо в консоль, например: 
```suggestion
EMAIL_BACKEND=django.core.mail.backends.console.EmailBackend
```
чтобы не "дергать" реальный сервер